### PR TITLE
Updated cal.py with new addition method

### PR DESCRIPTION
**Pull request by yaubourg2**
Updated cal.py with addingTwoNumbers method

### CODE REVIEW

**Review by Obinnajude — CHANGES_REQUESTED** [review]
Hello Yves! Thanks for this great contribution. You just have to make a few changes for your request to be merged.

The addition of two numbers function has been implemented. You can do the one for adding three numbers. 

You don't need to create a function or request input. That has been done for you. All you have to do is to do the addition using the numb, numb1 and numb3 arguments and print the answer

Thanks, expecting your new request

**Comment by yvesaubourg** [comment]
Hi Jude. As per your request, I have modified the code to calculate the addition of three numbers.

**Review by Obinnajude — CHANGES_REQUESTED** [review]
Hello Yves, Your code for adding and printing three numbers is very right, but you are not supposed to define the function yourself.

Read through the code and you will see a function defined already called AddThreeNumbers(numb,numb2)
Inside the function, you will see where it asked you to add your name and your code.
Enter your name in the print statement and repeat only your line of code where you print(numb,"+" numb1 "+" numb3) then your answer as you did before.

Once that is done, I will merge your code. Rembere to add your name. I can't merge it now, as it will break the code
Thanks for your patience.

**Pull request closed without merging**
